cannon/mipsevm/testutil/elf.go

```diff
@@ -10,8 +10,22 @@ import (
 )
 
 func LoadELFProgram[T mipsevm.FPVMState](t require.TestingT, name string, initState program.CreateInitialFPVMState[T], doPatchGo bool) T {
```

Comment:
Should we just cut this method altogether?

Reply:
indeed. fixed.